tests/channels/pjsip/identify/request_uri/configs/ast1/pjsip.conf

```diff
@@ -0,0 +1,35 @@
+[global]
+type=global
+debug=yes
+
+[system]
+type=system
+timer_t1=100
+timer_b=6400
+
+[local-transport-template](!)
+type=transport
+bind=127.0.0.1
+
+[local-transport-udp](local-transport-template)
+protocol=udp
+
+[endpoint-template-ipv4](!)
+type=endpoint
+context=default
+allow=!all,ulaw,alaw
+media_address=127.0.0.1
+identify_by=request_uri
+
+[alice](endpoint-template-ipv4)
+
+[identify-template](!)
+type=identify
+
+[alice-identify-literal](identify-template)
+endpoint=alice
+match_request_uri=sip:echo@127.0.0.1:5060\;param=yep
+
+[alice-identify-regex](identify-template)
+endpoint=alice
+match_request_uri=/long.*exp/
```

Comment:
You don't need to use the templates for a config this simple.  No need to change unless you want to though.  This is more for future reference.